### DISCUSSION TITLE
Welcome to BrewUNO Discussions!

### DISCUSSION THREAD

**Discussion by uncodead** [uncodead/BrewUNO#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsvcosta** [comment]
Olá gostaria de saber se é obrigatório para montar o brewuno ter uma interface ou posso fazer sem ela

**Comment by CarlosPedroGervasioJunior** [comment]
Bom dia a todos , obrigado por disponibilizar o projeto ! Gostaria de saber se consigo gravar o binario em um ESP32 Devkitv1 e se , como o @jsvcosta  perguntou , se posso montar sem LCD e teclado , usando apenas a interface WEB do BrewUno.

**Comment by 1337lucas** [comment]
@jsvcosta @CarlosPedroGervasioJunior Conseguiu fazer sem a parte de tela lcd? Foi a primeira coisa que pensei aqui tambem. Remover tela, botao e deixar mais simples e controlar apenas pela interface web.
